docs/how-to/add-delete-user-entity.md

```diff
@@ -0,0 +1,65 @@
+---
+id: add-delete-user-entity
+title: How To Add and Delete The User Entity
+sidebar_label: Add and Delete The User Entity
+slug: /how-to/add-delete-user-entity
+pagination_next: getting-started/authentication
+---
+
+# How To Add, Delete, and Change The User Entity
+
+The _User_ entity **is required** in order for you to enable authentication on your service.
+
+If you choose to [create a service _without_ authentication](/authentication/#how-to-create-a-service-without-authentication) then the _User_ entity will not be created.
+
+## Adding The User Entity
+
+First, go into your service's _Entities_ page.
+Click on the **Add Entity** button in the top right hand corner.
+Type `User` into the entity name text field and then click on the **Create Entity** button.
+
+![Add a new User entity to your Amplication service](./../getting-started/assets/authentication/new_user_entity.png)
+
+A popup will appear and give you additional context on restoring the _User_ entity.
+Click on the **Restore Default** button and the original _User_ entity provided by Amplication will be restored.
+
+:::note
+You have to add the _User_ entity first and then enable the [_NestJS Auth Module_ plugin](/authentication/#nestjs-auth-module-mandatory).
+It's not possible to do this in reverse order.
+:::
+
```

Comment:
@dericksozo lets add a note that when you try to install the auth plugin without the user entity you will be offered to create one
![image](https://github.com/amplication/docs/assets/112330500/0d174963-37f7-42a1-9996-9dfbb1b5165f)

Reply:
Taken care of in [this commit](https://github.com/amplication/docs/pull/315/commits/0625e3879f1480c31feb4b39eafbad5e8881fe17).